no matching function for call to 'max(votca::Index, long int)'

On i366:
```
/builddir/build/BUILD/votca-1.6_rc2/xtp/include/votca/xtp/checkpointreader.h: In member function 'void votca::xtp::CheckpointReader::ReadData(const CptLoc&, std::vector<std::__cxx11::basic_string<char> >&, const string&) const':
/builddir/build/BUILD/votca-1.6_rc2/xtp/include/votca/xtp/checkpointreader.h:235:35: warning: conversion from 'hsize_t' {aka 'long long unsigned int'} to 'std::vector<char*>::size_type' {aka 'unsigned int'} may change value [-Wconversion]
  235 |     std::vector<char*> temp(dims[0]);
      |                             ~~~~~~^
/builddir/build/BUILD/votca-1.6_rc2/xtp/include/votca/xtp/checkpointreader.h:247:21: warning: conversion from 'hsize_t' {aka 'long long unsigned int'} to 'std::vector<std::__cxx11::basic_string<char> >::size_type' {aka 'unsigned int'} may change value [-Wconversion]
  247 |     v.reserve(dims[0]);
      |               ~~~~~~^
/builddir/build/BUILD/votca-1.6_rc2/xtp/include/votca/xtp/checkpointreader.h: In member function 'void votca::xtp::CheckpointReader::ReadData(const CptLoc&, std::vector<Eigen::Matrix<double, 3, 1> >&, const string&) const':
/builddir/build/BUILD/votca-1.6_rc2/xtp/include/votca/xtp/checkpointreader.h:270:37: warning: conversion from 'hsize_t' {aka 'long long unsigned int'} to 'size_t' {aka 'unsigned int'} may change value [-Wconversion]
  270 |     size_t count = parent.getNumObjs();
      |                    ~~~~~~~~~~~~~~~~~^~
/builddir/build/BUILD/votca-1.6_rc2/xtp/src/libxtp/grids/vxc_grid.cc: In member function 'votca::Index votca::xtp::Vxc_Grid::UpdateOrder(votca::xtp::LebedevGrid&, votca::Index, std::vector<double>&, double) const':
/builddir/build/BUILD/votca-1.6_rc2/xtp/src/libxtp/grids/vxc_grid.cc:182:77: error: no matching function for call to 'max(votca::Index, long int)'
  182 |           sphericalgridofElement.getOrderFromIndex(std::max(maxindex - 1, 4l));
      |                                                                             ^
```

Build log here: 
[build_i386.txt](https://github.com/votca/xtp/files/4182135/build_i386.txt)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,9 @@
 For more detailed information about the changes see the history of the
 [repository](https://github.com/votca/xtp/commits/master).
 
+## Version 1.6 (released XX.02.20)
+* fix 32-bit build (#381, #380)
+
 ## Version 1.6_rc2 (released 10.02.20)
 * fix remove giant logo from tarball (#337)
 * fix assertions related to GLIBCXX_ASSERTIONS (#345)

--- a/src/libxtp/grids/vxc_grid.cc
+++ b/src/libxtp/grids/vxc_grid.cc
@@ -178,13 +178,15 @@ Index Vxc_Grid::UpdateOrder(LebedevGrid& sphericalgridofElement, Index maxorder,
     } else if ((r >= PruningIntervals[0]) && (r < PruningIntervals[1])) {
       order = sphericalgridofElement.getOrderFromIndex(4);
     } else if ((r >= PruningIntervals[1]) && (r < PruningIntervals[2])) {
+      constexpr Index maximum = 4;
       order =
-          sphericalgridofElement.getOrderFromIndex(std::max(maxindex - 1, 4l));
+          sphericalgridofElement.getOrderFromIndex(std::max(maxindex - 1, maximum));
     } else if ((r >= PruningIntervals[2]) && (r < PruningIntervals[3])) {
       order = maxorder;
     } else {
+      constexpr Index maximum = 1;
       order =
-          sphericalgridofElement.getOrderFromIndex(std::max(maxindex - 1, 1l));
+          sphericalgridofElement.getOrderFromIndex(std::max(maxindex - 1, maximum));
     }
   }
   return order;

--- a/src/libxtp/grids/vxc_grid.cc
+++ b/src/libxtp/grids/vxc_grid.cc
@@ -179,14 +179,14 @@ Index Vxc_Grid::UpdateOrder(LebedevGrid& sphericalgridofElement, Index maxorder,
       order = sphericalgridofElement.getOrderFromIndex(4);
     } else if ((r >= PruningIntervals[1]) && (r < PruningIntervals[2])) {
       constexpr Index maximum = 4;
-      order =
-          sphericalgridofElement.getOrderFromIndex(std::max(maxindex - 1, maximum));
+      order = sphericalgridofElement.getOrderFromIndex(
+          std::max(maxindex - 1, maximum));
     } else if ((r >= PruningIntervals[2]) && (r < PruningIntervals[3])) {
       order = maxorder;
     } else {
       constexpr Index maximum = 1;
-      order =
-          sphericalgridofElement.getOrderFromIndex(std::max(maxindex - 1, maximum));
+      order = sphericalgridofElement.getOrderFromIndex(
+          std::max(maxindex - 1, maximum));
     }
   }
   return order;

